### PR TITLE
some grammatical errors

### PR DESCRIPTION
**Pull request by wxb0521**
Signed-off-by: Xinbo Weng <xihuanbo_0521@zju.edu.cn>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: experimental/docker-stacks-and-bundles.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ are persisted as files, the file extension is `.dab` (Docker 1.12RC2 tools use
 A bundle has two top-level fields: `version` and `services`. The version used
 by Docker 1.12 tools is `0.1`.
 
-`services` in the bundle are the services that comprise the app. They
+`services` in the bundle is the services that comprise the app. They
```
Comment (thaJeztah) [inline experimental/docker-stacks-and-bundles.md]: this change doesn't look correct

File: image/spec/v1.1.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ This specification uses the following terms:
         <code>my-app:3.1.4</code>, <code>my-app</code> is the <i>Repository</i>
         component of the name. A repository name is made up of slash-separated
         name components, optionally prefixed by a DNS hostname. The hostname
-        must follow comply with standard DNS rules, but may not contain
+        must comply with standard DNS rules, but may not contain
```
Comment (thaJeztah) [inline image/spec/v1.1.md]: /cc @aaronlehmann

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks; found one issue, and /cc'd @aaronlehmann for the change in the image-spec

File: experimental/docker-stacks-and-bundles.md (hunk 1 above)
Comment (wxb0521) [inline experimental/docker-stacks-and-bundles.md]: I think `services`  is a field name , like a proper noun. So it should be considered singular.

Comment (thaJeztah) [inline experimental/docker-stacks-and-bundles.md]: I understand, but the sentence is hard to read this way. Consider changing to 

"services in the bundle is a list of service that comprise the app"

"in the bundle"  may not be needed, as the introduction above is already mentioning this is about the bundle file.

Comment (wxb0521) [inline experimental/docker-stacks-and-bundles.md]: Thank you! I understand your meaning. And `are` is correct.

File: image/spec/v1.1.md (hunk 2 above)
Comment (thaJeztah) [inline image/spec/v1.1.md]: My preference would go to "comply with" as well, but wanted to check with @aaronlehmann, the devil may be in the detail with specs

Comment (aaronlehmann) [inline image/spec/v1.1.md]: Either one is fine.

**Comment by vdemeester** [comment]
ping @wxb0521 could you take care of @thaJeztah comment ? 👼

**Comment by AkihiroSuda** [comment]
ping @wxb0521

**Comment by LK4D4** [comment]
@thaJeztah mind to carry this?

**Comment by thaJeztah** [comment]
carried in https://github.com/docker/docker/pull/30620

**Pull request closed without merging**